css2/cover.src

```diff
@@ -78,39 +94,32 @@
 
 <h2 id=abstract>Abstract</h2> 
 
-<p>This specification defines Cascading Style Sheets, level&nbsp;2
-revision&nbsp;1 (CSS&nbsp;2.1). CSS&nbsp;2.1 is a style sheet language
+<p>This specification defines Cascading Style Sheets level&nbsp;2 (CSS&nbsp;2)
+revision&nbsp;2 (CSS&nbsp;2.2). CSS is a style sheet language
 that allows
 authors and users to attach style (e.g., fonts and spacing) to
 structured documents (e.g., HTML documents and XML
 applications). By separating the presentation style of documents from
-the content of documents, CSS&nbsp;2.1 simplifies Web authoring and
+the content of documents, CSS simplifies Web authoring and
 site maintenance.
 
-<P>CSS&nbsp;2.1 builds on CSS2 [[-CSS2]] which builds on CSS1 [[-CSS1]].
+<p>
 It supports media-specific style sheets so that authors may tailor the
 presentation of their documents to visual browsers, aural devices,
 printers, braille devices, handheld devices, etc. It also supports
 content positioning, table layout, features for internationalization
 and some properties related to user interface.
 
-<p>CSS&nbsp;2.1 corrects a few errors in CSS2 (the most important
-being a new definition of the height/width of absolutely positioned
-elements, more influence for HTML's "style" attribute and a new
-calculation of the 'clip' property), and adds a few highly requested
-features which have already been widely implemented.
-But most of all CSS&nbsp;2.1
-represents a "snapshot" of CSS usage: it consists of all CSS features
-that are implemented interoperably at the date of publication of the Recommendation.
-
-<p>CSS&nbsp;2.1 is derived from and is intended to replace CSS2. Some
-parts of CSS2 are unchanged in CSS&nbsp;2.1, some parts have been
-altered, and some parts removed. The removed portions may be used in a
-future CSS3 specification. Future specs should refer to CSS&nbsp;2.1
-(unless they need features from CSS2 which have been dropped in
-CSS&nbsp;2.1, and then they should only reference CSS2 for those
-features, or preferably reference such feature(s) in the respective
-CSS3 Module that includes those feature(s)).
+<p>CSS&nbsp;2.2 is derived from and is intended to replace the
+CSS&nbsp;2.1 and CSS2&nbsp;(1998)
+specifications. See <a href="changes.html">appendix C of this
+document</a> for changes from CSS&nbsp;2.1,
+and <a href="/TR/2011/REC-CSS21-2011-06-07/changes.html">appendix C of
+CSS&nbsp;2.1 for changes from CSS2&nbsp;(1998)</a> (noting that certain CSS2&nbsp;(1998) features
+were removed from CSS&nbsp;2 in CSS&nbsp2.1). All future
```

Comment:
Missing semicolon in the latter nbsp.

Reply:
There's lots of entities that are missing semicolons. :( Because, y'know, it's valid SGML and software should cope. Or something.